src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function getLowerCase(str: string, name: string) {
+  return str.toLowerCase().trim().includes(name.toLowerCase().trim());
+}
+
 export const App: React.FC = () => {
+  const [nameMovies, setSearch] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(list => (
```

Comment:
1. not a list, it's a single movie
2. even so, it's better to use decomposing structure
```suggestion
  const visibleMovies = moviesFromServer.filter(({ title, description })=> (
```